typescript/deploy/src/router/deploy.ts

```diff
@@ -57,6 +55,24 @@ export abstract class AbacusRouterDeployer<
         }
       }),
     );
+  }
+
+  async transferOwnership(contractsMap: ChainMap<Chain, Contracts>) {
+    this.logger(`Transfer Ownership of routers to owner ...`);
+    await promiseObjAll(
+      objMap(contractsMap, async (chain, contracts) => {
```

Comment:
The (small) problem with this objMap util we seem to use in many places is that it's not immediately evident if it's iterating on the object values or keys